enigma-swing/src/main/java/cuchaz/enigma/gui/panels/IdentifierPanel.java

```diff
@@ -79,6 +80,36 @@ private void onModifierChanged(AccessModifier modifier) {
 	public void refreshReference() {
 		this.deobfEntry = entry == null ? null : gui.getController().project.getMapper().deobfuscate(this.entry);
 
+		// Prevent IdentifierPanel from being rebuilt if you didn't click off.
+		if (this.lastEntry == this.entry && this.nameField != null) {
+			if (!this.nameField.hasChanges()) {
+
+				final String name;
+
+				// Find what to set the name to.
+				if (this.deobfEntry instanceof MethodEntry && ((MethodEntry) this.deobfEntry).isConstructor()) {
```

Comment:
instance variable here please! (`instanceof MethodEntry methodEntry`)

Reply:
Not sure why I thought that it wasn't allowed; probably the codestyle looked too Java 8-like.

Lemme change that part around.